Bootcamp2015.AmazingRace.Base/Helpers/BackgroundTaskHelpers.cs

```diff
@@ -0,0 +1,49 @@
+﻿using System;
+using System.Linq;
+using System.Threading.Tasks;
+using Windows.ApplicationModel.Background;
+
+namespace Bootcamp2015.AmazingRace.Base.Helpers
+{
+    public class BackgroundTaskHelpers
+    {
+        private const string BT_ENTRY = "Bootcamp2015.AmazingRace.BackgroundTask.BTask";
+        private const string BT_NAME = "Bootcamp Amazing Race BTask";
+
+        public static async Task<bool> BackgroundTaskRegister()
+        {
+            // Check if the task is already registered
+            if (BackgroundTaskRegistration.AllTasks.Any(t => t.Value.Name == BT_NAME))
+                return true;
+
+            // Request access -> return false if denied
+            var res = await BackgroundExecutionManager.RequestAccessAsync();
+            if (res == BackgroundAccessStatus.Denied)
+                return false;
+
+            // Set up the trigger
+            var trigger = new TimeTrigger(15, false);
```

Comment:
Another option would be to use geofencing which can update when a user leaves a specific area and happens more frequently since the time trigger will update at most every 15 minutes.